ct.yaml

```diff
@@ -0,0 +1,9 @@
+# See https://github.com/helm/chart-testing#configuration
+remote: origin
+chart-dirs:
+  - install/kubernetes
+all: true
+debug: true
+validate-maintainers: false
+check-version-increment: false
+helm-extra-args: --timeout 10m0s
```

Comment:
minor nit, is this file required to be in the root of the repo?

In the interests of minimizing the sheer number of files in this directory, it'd be nice to move it somewhere related to github actions; also my first impression of `ct` in the context of Cilium was not "chart testing", but "conntrack" which I found surprising ;-)

Reply:
You are right, it's not required to be in root directory. As suggested by @nebril, I thinking _install/kubernetest_ dir is a good place to put. Let me know if you have other idea.

Reply:
I also renamed ct.yaml to chart-testing.yaml to avoid above confusion, thanks